src/main.c

```diff
@@ -2757,6 +2844,9 @@ int craft_main(int argc, char **argv) {
         double last_update = glfwGetTime();
         GLuint sky_buffer = gen_sky_buffer();
 
+        bool teleportButtonPressedLastFrame = false;
+        bool crouchingButtonPressedLastFrame = false;
```

Comment:
This Variable is also massive and should have a shorter name even if its slightly less descriptive.